app-localize-behavior.html

```diff
@@ -96,20 +96,6 @@
     ajax: null     /* Global iron-ajax object used to request resource files. */
   },
 
-  /**
```

Comment:
Oh I see, it mentions `pathToResources`. I would update the description, then, to say that it's fire after the resources have been loaded.